apstra/two_stage_l3_clos_client.go

```diff
@@ -645,6 +645,25 @@ func (o *TwoStageL3ClosClient) GetIbaWidget(ctx context.Context, id ObjectId) (*
 	return rawWidget.polish()
 }
 
+// GetAllIbaDashboards returns a list of IBA Dashboards in the blueprint
+func (o *TwoStageL3ClosClient) GetAllIbaDashboards(ctx context.Context) ([]IbaDashboard, error) {
+	rawDashes, err := o.client.getAllIbaDashboards(ctx, o.blueprintId)
+	if err != nil {
+		return nil, err
+	}
+
+	dashes := make([]IbaDashboard, len(rawDashes))
+	for i, w := range rawDashes {
+		pw, err := w.polish()
+		dashes[i] = *pw
+		if err != nil {
+			return dashes, err
+		}
+	}
```

Comment:
Please move err check to immediately after the error is generated.

Please return `nil` in the error case.